pandas/core/generic.py

```diff
@@ -5447,27 +5447,18 @@ def f():
 
         self._protect_consolidate(f)
 
-    def _consolidate(self, inplace: bool_t = False):
+    def _consolidate(self):
```

Comment:
do we actually need this method? its private, ok to remove .

Reply:
havent looked since i dont see the non-inplace consolidation as a problem